controllers/membersController.js

```diff
@@ -21,6 +22,34 @@ const getMembers = async (req, res) => {
   }
 }
 
+/**
+ * Fetches the data about our inactive/idle members
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getIdleMembers = async (req, res) => {
+  try {
+    let allMemberUsernames = await memberQuery.fetchMemberUsernames()
+    allMemberUsernames = allMemberUsernames.map(member => member.username)
+
+    let taskParticipants = await tasks.fetchActiveTaskMembers()
+    taskParticipants = [...new Set(taskParticipants)]
+
+    const idleMemberUserNames = allMemberUsernames.filter(member => taskParticipants.indexOf(member) === -1)
```

Comment:
I think instead of destruturing the `sets` back to an array, you can simply make the full of `Sets` like this below

```javascript
    taskParticipants = new Set(taskParticipants)
    const idleMemberUserNames = allMemberUsernames.filter(member => !taskParticipants.has(member))
```